src/CMakeLists.txt

```diff
@@ -177,6 +178,11 @@ add_custom_command(OUTPUT zitopt.h
     DEPENDS "${CMAKE_CURRENT_BINARY_DIR}/zitopt.ggo"
 )
 
+add_custom_command(OUTPUT ztopt.h
+	COMMAND gengetopt -C --no-help --no-version --unamed-opts=SUBNETS -i "${CMAKE_CURRENT_BINARY_DIR}/ztopt.ggo" -F "${CMAKE_CURRENT_BINARY_DIR}/ztopt"
```

Comment:
`--unamed-opts` should probably be excluded, since I don't think ztest takes arguments.